applications/CoSimulationApplication/python_scripts/base_classes/co_simulation_coupled_solver.py

```diff
@@ -245,13 +250,10 @@ def Check(self):
             coupling_operation.Check()
 
     def __CreateSolverWrappers(self):
-        ### ATTENTION, big flaw, also the participants can be coupled solvers !!!
```

Comment:
@adityaghantasala btw this comment is outdated, using coupled solvers also works now